src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/MsQuicStream.cs

```diff
@@ -220,6 +238,7 @@ internal override ValueTask WriteAsync(ReadOnlySequence<byte> buffers, Cancellat
 
         internal override async ValueTask WriteAsync(ReadOnlySequence<byte> buffers, bool endStream, CancellationToken cancellationToken = default)
         {
+            // CR: we should propagate exceptions up here for write state.
```

Comment:
@wfurt I don't exactly remember what the problem was, could you remind me before I create an issue?